changelogs/fragments/1036-redis-cache-keyset-name.yaml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - redis - make the redis cache keyset name configurable
```

There's also a module called `redis`, and usually if not specified any closer, the module is meant. So let's better add the plugin type here to not confuse people :)
```suggestion
  - redis cache plugin - make the redis cache keyset name configurable
```